tools/dhtnode.cpp

```diff
@@ -29,7 +29,7 @@ extern "C" {
 using namespace dht;
 
 void print_usage() {
-    std::cout << "Usage: dhtnode [-p local_port] [-b bootstrap_host:port]" << std::endl << std::endl;
+    std::cout << "Usage: dhtnode [-h] [-v [logfile]] [-i] [-d] [-n network_id] [-p local_port[:port]] [-b [bootstrap_host]]" << std::endl << std::endl;
```

Comment:
Yes. The former is a mistake. 

The latter is a choice, really. Usage doesn't imply "normal usage" IMO, but is meant to show *all* available usage syntaxes. If you run `make --help` for instance, you'll see `Usage: make [options] [target] ...` with the listed options where `-h` appears. But I don't mind. I can remove it.